packages/perseus-core/src/index.ts

```diff
@@ -130,7 +130,10 @@ export {default as getCategorizerPublicWidgetOptions} from "./widgets/categorize
 export {default as getCSProgramPublicWidgetOptions} from "./widgets/cs-program/cs-program-util";
 export {default as getExpressionPublicWidgetOptions} from "./widgets/expression/expression-util";
 export {default as getGrapherPublicWidgetOptions} from "./widgets/grapher/grapher-util";
-export {default as getInteractiveGraphPublicWidgetOptions} from "./widgets/interactive-graph/interactive-graph-util";
+export {
+    default as getInteractiveGraphPublicWidgetOptions,
+    InteractiveGraphPublicWidgetOptions,
```

Comment:
I could be wrong, but for some reason I was thinking we needed `export type` for types specifically in `index.ts` files because of some weirdness with how things get exported from packages.

Reply:
Good call! This export broke Storybook, and changing it to a type export fixed it.